src/Extensions/MemberExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace SilverStripe\MFA\Extensions;
+
+use SilverStripe\MFA\Model\AuthenticationMethod;
+use SilverStripe\ORM\DataExtension;
+
+/**
+ * Extend Member to add relationship to authentication methods and track some specific preferences
+ *
+ * @package SilverStripe\MFA
```

Comment:
\@package tags are kind of redundant now that we have namespaces, may as well remove it